overrides-c9s.yaml

```diff
@@ -3,10 +3,13 @@
 # we need in the `packages` section. When not needed. Empty or comment out this
 # file (except this comment).
 
-repos:
-  - c9s-baseos-mirror
-  - c9s-appstream-mirror
+#repos:
+#  - c9s-baseos-mirror
+#  - c9s-appstream-mirror
 
-packages:
+#packages:
   # https://github.com/openshift/os/issues/1514
-  - selinux-policy-38.1.36-1.el9
+  # For now we build with the latest selinux as this package is not available in the
+  # mirrors anymore. This needs to be fixed before we create c9s bootimages, for CI
+  # to pass.
+  #  - selinux-policy-38.1.36-1.el9
```

Comment:
You can just nuke all these lines and only leave `#packages:`.

Reply:
done.